src/data/finset/powerset.lean

```diff
@@ -222,17 +222,28 @@ begin
     simp }
 end
 
-lemma powerset_card_bUnion [decidable_eq (finset α)] (s : finset α) :
-  finset.powerset s = (range (s.card + 1)).bUnion (λ i, powerset_len i s) :=
+lemma pairwise_disjoint_powerset_len (s : finset α) :
+  _root_.pairwise (λ i j, disjoint (s.powerset_len i) (s.powerset_len j)) :=
+λ i j hij, finset.disjoint_left.mpr $ λ x hi hj, hij $
+  (mem_powerset_len.mp hi).2.symm.trans (mem_powerset_len.mp hj).2
+
+lemma powerset_card_disj_Union (s : finset α) :
```

Comment:
I guess `powerset_card_bUnion` is already a name, but this is a bit poor. I would prefer something like 
`powerset_eq_disj_Union` or even `powerset_eq_disj_Union_powerset_len`.

Reply:
Happy for me to leave that as a follow-up?  I don't really want to stretch "reprove bUnion lemmas in terms of disj_union" to also "clean up some weird lemma names"

Reply:
Fine by me.